css/main.css

```diff
@@ -1,3 +1,673 @@
-/* Add your CSS here */
+:root {
 
-/* Dont' forget to link this file to your HTML in the <head> */
+    /* colors */
+    --black: #000000;
+    --white: #ffffff;
+    --lilac: #f5e8ff;
+    --light-green: #E6FAC0;
+    --medium-green: #5E7D1F;
+    --dark-green: #394B2A;
+    
+    /*  fw-fonts-weight */
+    --fw-400: 400;
+    --fw-500: 600;
+    --fw-700: 700;
+    --fw-900: 900;
+   
+    /* fs-font-size */
+    
+
+    --fs-2px:   0.125rem;
+    --fs-12px:  0.75rem;
+    --fs-14px:  0.875rem;
+    --fs-15px:  0.938rem;
+    --fs-20px:  1.25rem;
+    --fs-24px:  1.5rem;
+    --fs-28px:  1.75rem;
+    --fs-32px:  2rem;
+    --fs-36px:  2.25rem;
+    --fs-200px: 12.5rem;
+    --fs-320px: 20rem;
+
+    /* other sizes */
+    --fm-1em:   1em;
+    --fm-2em:   2em;
+    --fm-4em:   4em;
+    --fm-16em: 16em;
+
+    --fm-1rem:   1rem;
+    --fm-1p2rem: 1.2rem;
+    --fm-1p3rem: 1.3rem;
+    --fm-1p4rem: 1.4rem;
+    --fm-1p6rem: 1.6rem;
+    --fm-1p8rem: 1.8rem;
+    --fm-3rem:   3rem;
+    --fm-5rem:   5rem;
+}
+
+
+/* RESET */
+
+*,
+html {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    outline: 0;
+    border: 0;
+}
+
+
+/* Fonts */
+body {
+    font-family: 'Roboto', sans-serif;   
+}
+
+
+.home {
```

Comment:
Do we really need a grid here? Each row looks like a normal container/element to me